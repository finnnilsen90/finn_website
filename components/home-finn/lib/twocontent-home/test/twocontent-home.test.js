const React = require('react');
const ReactTestUtils = require('react-dom/test-utils');
const renderer = require('react-test-renderer')
let { shallow, mount, render } = require('enzyme');

const Twocontent = require('../twocontent-home.js');
const Slider = require('../../slider-home/slider-home.js');
const Button = require('../../../../lib/button-one/button-one.js');

let newTwocontent = new Twocontent();

describe('component connection', () => {

    it('should be defined', () => {
        expect(Twocontent).toBeDefined();
    });

    it('slider should be defined', () => {
        expect(Slider).toBeDefined();
    });

    it('button should be defined', () => {
        expect(Button).toBeDefined();
    });

});  
    
describe('<Twocontent />', () => {

    const wrapper = render(<Twocontent />);

    it('renders components', () => {
        expect(wrapper).toMatchSnapshot();
    });

    it('button should say Resume', () => {
        expect(wrapper.find('.resume_btn')).toMatchSnapshot()
    })


})