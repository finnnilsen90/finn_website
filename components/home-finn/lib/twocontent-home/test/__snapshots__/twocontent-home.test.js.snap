// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Twocontent /> button should say Resume 1`] = `null`;

exports[`<Twocontent /> renders components 1`] = `
<div
  class="twocontent-home_Container font"
>
  <h1
    class="resume_header"
  >
    <a
      href="/resume"
    >
      Resume
    </a>
  </h1>
  <img
    class="resume_icon"
    src="test-file-stub"
  />
  <div
    class="slider-home_Container font"
  >
    <div
      class="outer"
    >
      <div
        class="slide1 slide"
      >
        <a
          href="/resume"
        >
          <h2>
            Associate Director, Media and Technology
          </h2>
          <div
            class="slider_class"
          >
            Mogo Interactive
          </div>
        </a>
      </div>
      <div
        class="slide2 slide"
      >
        <a
          href="/resume"
        >
          <h2>
            Product Manager
          </h2>
          <div
            class="slider_class"
          >
            Mogo Interactive
          </div>
        </a>
      </div>
      <div
        class="slide3 slide"
      >
        <a
          href="/resume"
        >
          <h2>
            ChatScript Developer
          </h2>
          <div
            class="slider_class"
          >
            JustAnswer.com
          </div>
        </a>
      </div>
    </div>
  </div>
</div>
`;
